### DISCUSSION TITLE
iphone - safari 썸네일 두 번째 다운로드부터 반응 없음

### DISCUSSION THREAD

**Discussion by da-in** [Yangjaecheon-Otter-Guardians/simple-thumbnail#118] [Ideas]
### 주제

Features

### Body

iphone - safari 썸네일 두 번째 다운로드부터,
**다운로드 버튼을 눌렀을 때 아무런 반응이 없어요.**
(iphone 12 pro, safari 기준)

Downloads 폴더에 들어가면 잘 저장되어있습니다.
그런데 첫 번째 다운로드와 다르게 다운로드가 되고 있다, 되었다는 반응이 없네요...

왜 그럴까요. 원인을 찾으면 해결되었으면 좋겠어요.
원인을 못찾으면 별도로 다운로드 폴더에 저장되었습니다. 팝업이라도 띄워야 안헤맬 것 같습니당....

### BDD TASK

- [ ] 다운로드 버튼을 누른다 > 다운로드 시작 혹은 종료와 관련한 반응이 나타난다.

**Comment by caseBread** [comment]
팝업 띄워주는 것 좋은 방안인 것 같습니다.

다음도 생각해보면 좋을 것 같아요

- iOS만 팝업을 띄워주는건 어떨가요
- toast방식처럼 자동으로 닫히는 팝업? 아니면 사용자가 닫기버튼을 눌러야되는지?

**Reply by da-in** [reply]
어랏 그런데 정확한 부검이 필요할 것 같습니다.

"다운로드 하시겠습니까?" 하는 브라우저 내장 팝업은 뜨고 그 이후 모션이 다른 것 같기도 해서
기기별로 테스트 해보고, 우리가 컨트롤 할 수 있는 영역인지도 추후 체크해보면 좋을 것 같습니다.🤔

**Reply by caseBread** [reply]
오 네 
[우리가 컨트롤할 수 있는 영역]인지 체크해보면 좋을 것 같아 저희와 비슷한 레퍼런스를 좀 찾아보았는데요,

https://jjalbang.today/jjalbangMake/13114

위 사이트를 보면, 동일조건에서 다운로드가 되더라구여,, 방법이 있긴 한가봅니다 🫠
